packages/firebase_core/firebase_core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.5.0-dev.2
```

Comment:
0.5.0-dev.1 is published, not sure if this is the version you want.

Reply:
0.5.0-dev.2 is fine, thanks!